src/amo/pages/Home/index.js

```diff
@@ -184,6 +183,23 @@ export class HomeBase extends React.Component {
     );
   }
 
+  renderHeroHeader() {
+    const { i18n } = this.props;
+
+    return (
+      <div className="Home-heroHeader">
+        <h2 className="Home-heroHeader-title">
+          {i18n.gettext('Extensions are like apps for your browser.')}
+        </h2>
+        <h3 className="Home-heroHeader-subtitle">
+          {i18n.gettext(
+            'They add features to Firefox to make browsing faster, smarter, or just plain fun.',
+          )}
+        </h3>
+      </div>
+    );
+  }
```

Comment:
Rather than hard-coding this, can't we use the secondary hero api response like we do on Desktop Firefox?

Reply:
> Rather than hard-coding this, can't we use the secondary hero api response like we do on Desktop Firefox?

This is a good idea which I hadn't thought of. Currently the wording is slightly different, which would cause some difficulty. On desktop the headline is "Extensions are like apps for Firefox.", while on android the headline is "Extensions are like apps for your browser."

@jvillalobos / @devaneymoz Is this distinction important, or perhaps just an artifact of these being developed at different times? Would we be okay with using the same text (one of the two above) for both `/firefox` and `/android`?

Reply:
I think `Extensions are like apps for Firefox.` should work fine for both sites.